examples/dmm/dmm.py

```diff
@@ -193,16 +193,19 @@ def model(self, mini_batch, mini_batch_reversed, mini_batch_mask,
             # first compute the parameters of the diagonal gaussian distribution p(z_t | z_{t-1})
             z_mu, z_sigma = self.trans(z_prev)
             # then sample z_t according to dist.Normal(z_mu, z_sigma)
-            with poutine.scale(None, annealing_factor * mini_batch_mask[:, t - 1:t]):
```

Comment:
Do we also need to change the notebook?

Reply:
i think we should hold off on cleaning up tutorials until we've finalized all the breaking 0.2 changes, semantics, etc

Reply:
Sounds good, as long as our tutorials test pass.